kevm-pyk/src/tests/unit/test_kevm.py

```diff
@@ -50,3 +52,52 @@ def test_kinner_to_hex(test_id: str, input: KInner, result: KInner) -> None:
     to_hex = KEVM.kinner_to_hex(input)
     # Then
     assert to_hex == result
+
+
+JUMPDESTS_DATA: Final = [
```

Comment:
Could we also please have a test in which we have concrete bytecode interspersed with symbolic buffers?

Reply:
Added more tests.